### PR TITLE
ISSUE-133 Refine a relevant IMAP scenario

### PR DESCRIPTION
**Pull request by chibenwa**
- [x] TODO tun it with ~100 users and see what happens
 - [x] Scale to system limits

### CODE REVIEW

**Comment by chibenwa** [comment]
Screenshot from 2022-04-24 15-15-04

**Review by Arsnael — COMMENTED** [review]
Read it :) Looks ok to me so far

**Comment by chibenwa** [comment]
16K users...

Screenshot from 2022-04-25 11-01-32

**Comment by chibenwa** [comment]
Screenshot from 2022-04-25 11-02-39

Stable at 1600 req.s ...

File: build.sbt [hunk 1]
```diff
@@ -32,4 +34,4 @@ scalacOptions in ThisBuild ++= Seq("-unchecked", "-deprecation", "-feature", "-l
 
 enablePlugins(GatlingPlugin)
 
-lazy val gatlingImap = ProjectRef(uri("https://github.com/linagora/gatling-imap.git"), "gatling-imap")
+lazy val gatlingImap = ProjectRef(uri("https://github.com/linagora/gatling-imap.git"), s"gatling-imap-${UUID.randomUUID().toString}")
```
Comment (Arsnael) [inline build.sbt]: I get the idea, but wouldn't that fill up the disk after a bunch of builds?

Comment (chibenwa) [reply build.sbt]: Trying stuff....

**Review by Arsnael — APPROVED** [review]

**Pull request merged**